hold_duration:      15  # Duration to wait inside docking bay
goal_tolerance_pos: 0.5 # Distance tolerance for arriving at goal pose
goal_tolerance_ang: 0.1 # Angular tolerance for arriving at goal yaw

default_thrust: 0.2 # Thrust value to use for docking procedure

pix_threshold: 13 # (px) Threshold for centering the image (note, img is 300x300)
pix_offset:    13 # (px) Offset center x position in image (note, img is 300x300)
